hooks/scene_operation_tk-nuke.py

```diff
@@ -39,10 +39,23 @@ def execute(self, *args, **kwargs):
         engine = self.parent.engine
         if hasattr(engine, "hiero_enabled") and engine.hiero_enabled:
             return self._hiero_execute(*args, **kwargs)
+        elif hasattr(engine, "studio_enabled") and engine.studio_enabled:
+            return self._studio_execute(*args, **kwargs)
         else:
             return self._nuke_execute(*args, **kwargs)
 
+    def _studio_execute(self, operation, file_path, **kwargs):
+        """
+        The Nuke Studio specific scene operations.
+        """
+        # Out of the box, we treat Nuke Studio just like Hiero, so we
+        # can just call through here.
+        return self._hiero_execute(self, operation, file_path, **kwargs)
+
     def _hiero_execute(self, operation, file_path, **kwargs):
+        """
+        The Hiero specific scene operations.
```

Comment:
missing param docs. i realize these are private, but since you're adding docstrings... :)